client/src/main/java/org/eclipse/hono/client/impl/AbstractRequestResponseClient.java

```diff
@@ -413,7 +413,7 @@ public final void setRequestTimeout(final long timoutMillis) {
      * @param delivery The handle for accessing the message's disposition.
      * @param message The response message.
      */
-    protected final void handleResponse(final ProtonDelivery delivery, final Message message) {
+    public final void handleResponse(final ProtonDelivery delivery, final Message message) {
```

Comment:
is there a particular reason why this method needs to become public?

Reply:
After I moved the abstract base classes (back) to the impl package: no, not anymore.
Previously some test classes needed the `public` modifier.
I will switch it back to `protected`.